moped-editor/src/views/projects/projectView/ProjectComments.js

```diff
@@ -91,6 +92,11 @@ const projectNoteTypes = ["", "Internal Note", "Status Update"];
 
 const ProjectComments = (props) => {
   const isStatusEditModal = props.modal;
+  // use currentPhaseId if passed down from ProjectSummaryStatusUpdate component,
+  // otherwise use data passed from ProjectView
+  const currentPhaseId =
+    props.currentPhaseId ??
+    props.data?.moped_project[0]?.moped_proj_phases[0]?.moped_phase.phase_id;
```

Comment:
ahh—i see now that `props.data` is being passed through the `ProjectView` component and can't easily be avoided. thanks for adding that `currentPhaseId` for `ProjectSummaryStatusUpdate` and adding the comment!

Reply:
yeah, i went back and forth on this multiple times - this was how i originally did it before i submitted my initial pr. thanks for helping me pick an approach!!

Reply:
+1 the comments and variable names make this super readable 🙏